code/modules/tables/interactions.dm

```diff
@@ -133,6 +131,7 @@
 	var/list/click_params = params2list(params)
 	//Center the icon where the user clicked.
 	if(!click_params || !click_params["icon-x"] || !click_params["icon-y"])
+		world << "[click_params]"
```

Comment:
Oh? Is this a debug message you forgot to remove?

Reply:
oh shit yes it is

Reply:
@LudwigVonChesterfield While many of the other comments here are minor issues (ill try to fix them though) this one in particular is a blocking issue. At this point it would be appropriate to mark a review as changes requested

Reply:
doing